### ISSUE TITLE
Webui不适配reactchat

### ISSUE THREAD

**Issue by zhanpengjie** [QwenLM/Qwen-Agent#205]
目前webui只对functioncall有适配，react无法格式化步骤，请问有计划适配吗

**Comment by BestXuan** [comment]
Yes, may I ask when WebUI will adapt to React? And currently, I have directly changed the assistant of groupchat to react to run. Will you update groupchat based on react in the future?
